Oqtane.Client/Modules/Controls/LocalizableComponent.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using Microsoft.AspNetCore.Components;
+using Microsoft.Extensions.Localization;
+using Oqtane.Shared;
+
+namespace Oqtane.Modules.Controls
+{
+    public class LocalizableComponent : ModuleControlBase
+    {
+        private IStringLocalizer _localizer;
+
+        [Parameter]
+        public string ResourceKey { get; set; }
+
+        protected string Localize(string name)
```

Comment:
Just a little note here, I had seen `T` is common name for translation, so can we use it instead?

Reply:
can you provide more detail on what you are asking?

Reply:
Just if we can rename `Localize` to `T` or just forget it

Reply:
My preference is to use Localize() as its much more explicit/intuitive - I would have no idea what T means.

Reply:
T for Translate, I'm with you, seems I need to add a check in every child component, cause we have 3 cases:

1. Localizable component and key is exist

2. Localizable component and key isn't exist

3. Non localizable component

I will do a final test before merge

Reply:
![Localization](https://user-images.githubusercontent.com/3237266/96498561-e0066380-1254-11eb-834e-f8eef932a719.png)

Everything works as except, regarding the above cases the result should be:

1. The translated text

2. `{ResourceKey}.{Property}`

3. Use the property value